wso2bps/kubernetes/wso2bps-manager-controller.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
We may also need to add default-controller and default service yaml files for the default profile.